File: Marvellous_LB/7-C++/program4.cpp
#include<iostream>
using namespace std;

class Arithmatic
{
    public:
    int Addition(int iVal1, int iVal2)
    {
        int iAns=iVal1+iVal2;

        return iAns;
    }
};

int main()
{
    int iNo1=0, iNo2=0 , iAns=0;

    cout<<"Enter first number : \n";
    cin>>iNo1;

    cout<<"Enter second number : \n";
    cin>>iNo2;

    Arithmatic obj;
    iAns=obj.Addition(iNo1,iNo2);

    cout<<"Addition is :"<<iAns<< "\n";

    return 0;
}

File: Marvellous_LB/8-JAVA/program4.java
import java.util.*;

public class program4 
{

    static int Addition(int i,int j)
    {
        int Sum=0;
        Sum=i+j;
        return Sum;

    }
    public static void main(String args[])
    {
        Scanner sobj = new Scanner(System.in);

        int iNo1=0,iNo2=0,iAns=0;

        System.out.println("Enter First number : ");
        iNo1=sobj.nextInt();

        System.out.println("Enter second number : ");
        iNo2=sobj.nextInt();

        iAns=Addition(iNo1,iNo2);

        System.out.println("Addition is : "+iAns);

    }
}